content/articles/fastify-svelte-app/index.md

```diff
@@ -0,0 +1,466 @@
+Fastify is a low-overhead backend web framework that is built on top of the Node.js runtime. Its popularity and attention rising to its lightweight design and the plugin ecosystem it comes with. In Fastify the idea is that everything is a plugin. This allows us to quickly encapsulate the functionality in our project as a plugin which can then be distributed and used in other projects. 
+
+### Goal
+
+In this Fastify and Svelte tutorial, we will build a CRUD app that fetches blog posts.  We will be:
+
+- Setting up a Fastify in a Node.js environment.
+
+- Defining the API routes for our Fastify backend.
+
+- Adding the requests validation.
+
+- Loading and using Fastify plugins.
+
+- Setting up a Svelte frontend application using `degit`.
+
+- Data fetching and building components in Svelte.
+
+### Prerequisites
+
+1. Basics of programming with the [JavaScript](https://www.w3schools.com/js/DEFAULT.asp) programming and comfortability of the [Node.js](https://nodejs.org/en/) environment.
+
+2. Have [Node.js](https://nodejs.org/en/) JavaScript runtime installed on your development system.
+
+3. An IDE or code editor such as [VS Code](https://code.visualstudio.com/download).
+
+4. A basic understanding of [REST APIs](https://en.wikipedia.org/wiki/Representational_state_transfer) will be helpful.
+
+5. A web browser such as [Google Chrome]((https://www.google.com/chrome/)) on your development machine.
+
+### Setting Up The Backend with Node.js and Fastify
+
+To start the project, create a folder and name it `fastify-svelte-app`. Inside this folder, we will create another folder for our svelte client and name it `svelte-client`. Our backend code will be at the root of the main `fastify-svelte-app` folder. This is the folder we will start for now. Open the terminal and run the command `npm init -y` to initialize the npm with default configurations while creating a `package.json` file to manages our dependencies.
```

Comment:
What is this code tutorial (use case) about? State your objective before this topic.